### ISSUE TITLE
System templates - endpoint trackers

### ISSUE THREAD

**Issue by juchowan** [CiscoDevNet/terraform-provider-sdwan#159]
DNS entry should look like this
- only has endpoint-dns name,
- interval, multiplier and threshold being configured from system defaults
- type interface is a vmanage default

![image](https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/79261946/4a261317-17b9-4487-93e1-96ed3c896393)

What happens now with following config:
- no configured interval, multiplier, threshold - should read vmanage default
- no explicit type - should use vmanage default interface type

![image](https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/79261946/0057f329-9f64-46b2-8921-826b12d0d40d)

Entry is not shown in the gui. If i explicitly configure type interface it will be shown in GUI but with incomplete config and no DNS type selected:
![image](https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/79261946/016abfdd-39fd-46f3-b397-eb6254095952)
I could configure defaults in the terraform code, but i believe it should work without it.

For URL entry also with no type interface explicitly configured it doesn't use default, configures empty value intead and doesn't display in gui. The 3 timers are also used as empty, not default. 
![image](https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/79261946/2317bd68-2222-4e59-9eb2-fd03643f035a)

For endpoint tracker groups we have no type configured:
![image](https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/79261946/d4dfb6eb-95b5-4af2-a429-e3880c6d043e)

Doing it from GUI shows that configured type is tracker-group. This is not allowed in the code
![image](https://github.com/CiscoDevNet/terraform-provider-sdwan/assets/79261946/dee3bc52-c186-4db2-bfac-26212a522518)

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/c5282244b3a5460d50c4d4c1b252bbb6dbab9b45

**Comment by danischm** [comment]
This should allow "tracker-group" type for endpoint tracker groups. The "default" values (like multiplier, etc.) still need to be configured explicitly. The "sig" object tracker must be configured like this:
```
    {
      object_number = 2
      sig           = "[\"global\"]"
    }
```

**Comment by danischm** [comment]
This should be fixed in the v0.3.6 release, can you confirm?

**Comment by juchowan** [comment]
the type is still empty if i don't explicitly set it to any value (it should be "interface" by default). This is also something that we need to address in the module?

**Comment by danischm** [comment]
Yes, we are not making use of default values at the provider level.
